src/components/home/Home.vue

```diff
@@ -18,10 +18,9 @@
     />
     <GoogleMaps class="col-start-3 col-span-6" />
     <Form class="col-end-13 col-span-4"></Form>
-    <div class="container">
-      <div class="grid grid-cols-12 items-center"></div>
-    </div>
+    <ResultList class="pr-4 h-96 overflow-y-scroll col-span-4 col-end-13" />
   </div>
+  <div class="m-20 grid justify-items-center"><Charts /></div>
```

Comment:
jak dodasz klase bezpośrednio do charts to nie zadziała ? Raczej nie trzeba tego w diva opakowywać